src/common/POLFields/DetailsFields/FieldAutomaticExport.js

```diff
@@ -1,21 +1,34 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import { FormattedMessage } from 'react-intl';
 import { Field } from 'react-final-form';
 
-import { Checkbox } from '@folio/stripes/components';
+import { Checkbox, InfoPopover } from '@folio/stripes/components';
+
+const FieldAutomaticExport = ({ isManualOrder = false, ...props }) => {
+  const label = (
+    <>
+      <FormattedMessage id="ui-orders.poLine.automaticExport" />
+      {isManualOrder && <InfoPopover content={<FormattedMessage id="ui-orders.poLine.manualPO.info" />} />}
+    </>
+  );
 
-const FieldAutomaticExport = () => {
   return (
     <Field
       component={Checkbox}
       fullWidth
-      label={<FormattedMessage id="ui-orders.poLine.automaticExport" />}
+      label={label}
       name="automaticExport"
       type="checkbox"
       vertical
       validateFields={[]}
+      {...props}
     />
   );
 };
 
+FieldAutomaticExport.propTypes = {
+  isManualOrder: PropTypes.bool.isRequired,
```

Comment:
If it's required why do you specify default value for it?

Reply:
Made it required out of habit) Thanks, I'll fix it